.jshintrc

```diff
@@ -54,6 +54,9 @@
    * =================
    */
 
+  // Don't require semicolons
+  "asi": true,
+
```

Comment:
wow. just remove this file and `.jshintignore`. we're using `standard` anyway, so these shouldn't be there